app/controllers/notes_controller.rb

```diff
@@ -11,7 +11,7 @@ class NotesController < ApplicationController
 
   before_action :lookup_user, :only => [:index]
   before_action :set_locale
-  around_action :web_timeout
+  around_action :web_timeout, :only => :show
```

Comment:
There was no code for timeouts on non-map layout pages. https://github.com/openstreetmap/openstreetmap-website/commit/a7f0de41e65f90c8a20e1a219a5b7ecf498628a7 enabled timeouts here without defining any timeout templates.

Reply:
Moved notes-related changes to #4647.

Reply:
Just because there wasn't a template defined doesn't mean the timeout didn't work and protect the site - it just means the user got a bad response when the timeout happened!